tests/unit/test_conf.py

```diff
@@ -225,3 +226,22 @@ def test_max_key_fee_from_args(self):
         args = parser.parse_args(['--max-key-fee', '1.0', 'BTC'])
         c = Config.create_from_arguments(args)
         self.assertEqual(c.max_key_fee, {'amount': 1.0, 'currency': 'BTC'})
+
+    def test_one_of_string(self):
+        parser = argparse.ArgumentParser()
+        Config.contribute_to_argparse(parser)
+
+        with self.assertRaises(ValueError):
+            no_vaid_values = OneOfString([], "no valid values", None)
+
+        with self.assertRaises(ValueError):
+            default_none = OneOfString(["a"], "invalid default", None)
+        with self.assertRaises(ValueError):
+            invalid_default = OneOfString(["a"], "invalid default", "b")
+
+        valid_default = OneOfString(["a"], "valid default", "a")
```

Comment:
this isn't used / the value isn't checked

Reply:
this is a non-error test case. it tests that if the default is in the valid values list, it does not raise

Reply:
You've already "tested" the passing case by adding OneOfString to the main `TestConfig` and you'll actually use it to set some value below so that's enough I think.